Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/AppUpdaterCommandFactory.cs

```diff
@@ -106,7 +107,7 @@ public IValidateLicenseCommand CreateValidateLicenseCommand(AppUpdaterContext co
         public ICheckDiskSpace CreateCheckDiskSpaceCommandForDiff(int versionId, AppUpdaterContext context, CancellationToken cancellationToken)
         {
             // get project biggest file size
-            long biggestFileSize = 0;
+            long biggestFileSize = 0, sizeProcessedFiles = 0;
```

Comment:
Try to keep one convention. `processedFilesSize`  sounds better. Or even `filesToProcessSize`.

Reply:
Done